### DISCUSSION TITLE
How to configure Rsbuild to build outputs in esm/cjs and other formats?

### DISCUSSION THREAD

**Discussion by zhangfisher** [web-infra-dev/rspack#7313] [Q&A]
使用rsbuild构建库时，如何配置rsbuild构建出esm/cjs等格式的产物？

找了一圈没找到相关的配置，只有一个`output.target='node'`

**Comment by hardfist** [comment]
等 rslib 开源后就可以便捷的实现这个功能了

**Comment by chenjiahan** [comment]
The Rslib repository is now public: https://github.com/web-infra-dev/rslib

**Comment by sytana** [comment]
> 使用rsbuild构建库时，如何配置rsbuild构建出esm/cjs等格式的产物？
> 
> 找了一圈没找到相关的配置，只有一个`output.target='node'`

兄弟找到解决方案了吗
